Bearded.Monads.Tests/AsyncExtensionTests.cs

```diff
@@ -23,6 +23,50 @@ public async void SelectMany_Happy()
             Assert.Equal(expected, actual);
         }
 
+        [Fact]
+        public async void SelectManyTry_Happy()
+        {
+            async Task<Try<int>> TryResult(int value)
+            {
+                return await Task.FromResult(value.AsTry());
```

Comment:
You don't need to `await` a `Task.FromResult`, if the method is async it'll wrap the `value.AsTry()` into a task automatically. I know resharper complains, but it's fine :)